utils/common/src/test/kotlin/ExtensionsTest.kt

```diff
@@ -375,13 +376,21 @@ class ExtensionsTest : WordSpec({
 
             assertSoftly {
```

Comment:
out of scope: It would imo be nice to replace `assertSoftly / forEach` with `reserved.forAll`, maybe in a separate commit.

Reply:
Thanks!